include/nuttx/net/netdev.h

```diff
@@ -161,13 +161,16 @@
  * headers
  */
 
-#define ETHBUF ((FAR struct eth_hdr_s *)&dev->d_buf[0])
+#define ETHBUF ((FAR struct eth_hdr_s *)\
```

Comment:
let's move to private header file inside net/?

Reply:
Done